src/common/util/game.ts

```diff
@@ -703,7 +722,9 @@ export function applyAbilities(state: w.GameState): w.GameState {
         // console.log(`Applying ability of ${obj.card.name} to ${ability.targets}`);
         ability.currentTargets = executeCmd(state, ability.targets, obj) as w.Target;
         const targets: w.Targetable[] = ability.currentTargets.entries;
-        targets.forEach(ability.apply);
+        targets.forEach((target) => {
```

Comment:
Why was this change needed?

Reply:
Oops, this was something I was looking into but ended up not doing, let me just revert.